avocado/utils/software_manager.py

```diff
@@ -286,6 +286,25 @@ def list_files(self, name):
         except process.CmdError:
             return []
 
+    def rpm_install(self, file_path):
+        """
+        Install the rpm file [file_path] provided.
+
+        :param file_path: Rpm file path.
```

Comment:
Please specify `return` information as well, here and in other functions.

Reply:
sure